src/components/Quill/resources/generateWebViewIndex.ts

```diff
@@ -21,15 +21,24 @@ export function generateWebViewIndex(
         <style>
           html,
           body {
+            height: 100%;
             margin: 0;
             padding: 0;
+          }
+          
+          .ql-wrapper {
+            display: flex;
+            flex-direction: column;
             height: 100%;
           }
-
+          
           .quill-editor {
-            height: 100%;
-            width: 100%;
-            border: 0px !important;
+            flex: 1 1 auto;
+            border: 0px;
+          }
+          
+          .ql-wrapper .ql-container {
```

Comment:
```suggestion
          .quill-wrapper .ql-container {
```

Dit is consisten met de `quill-editor` die we al hebben.